CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.5.1 (2023-07-18)
+
+- Fixed `-User` parameter of `Add-TeamViewerUserGroupMember` and `Remove-TeamViewerUserGroupMember` to work for single User ID inputs.
+
```

Comment:
Otional: Probably something like "- Improved User ID parameter handling for `Add-TeamViewerUserGroupMember` and `Remove-TeamViewerUserGroupMember`" would be fine here, too.